composer.json

```diff
@@ -47,7 +47,7 @@
         "zendframework/zend-serializer": "^2.8",
         "zendframework/zend-session": "^2.7.3",
         "zendframework/zend-view": "^2.8.1",
-        "zendframework/zend-mvc-console": "^1.1"
+        "zendframework/zend-mvc-console": "^1.1.8"
```

Comment:
This doesn't look like a fix here: if `<1.1.8` is invalid for consumers, then it should be specified directly downstream in consumers.

Reply:
To clarify: if a downstream project relies on a specific dependency set and fails with lowest dependency versions, then it should simply bump this one dependency in its own definitions.

Reply:
I don't disagree but the error is very much outside the scope of what I think zend-mvc-console does.  Nevertheless, by uping the version to ^1.1.8 it fixes the following unit test failures.  And this is in the require-dev section.

```
...............E.E................                                34 / 34 (100%)

Time: 7.4 seconds, Memory: 22.00Mb

There were 2 errors:

1) DoctrineMongoODMModuleTest\Service\ConnectionFactoryTest::testConnectionStringShouldAllowUnixSockets
MongoDB\Driver\Exception\InvalidArgumentException: Failed to parse MongoDB URI: 'mongodb:///tmp/mongodb-27017.sock'. Invalid host string in URI.

/var/www/vendor/mongodb/mongodb/src/Client.php:56
/var/www/vendor/alcaeus/mongo-php-adapter/lib/Mongo/MongoClient.php:95
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:282
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:434
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:283
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:358
/var/www/tests/DoctrineMongoODMModuleTest/AbstractTest.php:44

2) DoctrineMongoODMModuleTest\Service\ConnectionFactoryTest::testDbNameShouldNotOverrideExplicitDefaultDB
MongoDB\Driver\Exception\InvalidArgumentException: Failed to parse MongoDB URI: 'mongodb://localhost:27017/test fails if this is defaultDB'. Invalid database name in URI.

/var/www/vendor/mongodb/mongodb/src/Client.php:56
/var/www/vendor/alcaeus/mongo-php-adapter/lib/Mongo/MongoClient.php:95
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:282
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:434
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:283
/var/www/vendor/doctrine/mongodb/lib/Doctrine/MongoDB/Connection.php:358
/var/www/tests/DoctrineMongoODMModuleTest/AbstractTest.php:44
```

Reply:
If it fixes tests in this package then 👍

I was confused by the explanation pointing to other packages